runtimes/protocol/notifications.ts

```diff
@@ -0,0 +1,73 @@
+import { MessageType, ProtocolNotificationType } from './lsp'
+
+export interface NotificationContent {
+    readonly text: string
+    readonly title?: string
+}
+
+export namespace FollowupNotificationActionType {
```

Comment:
Why `FollowupNotificationActionType` should be separate namespace from `NotificationActionType`?

Reply:
It's expected to be used for `NotificationFollowupParams`.